app/assets/stylesheets/application.scss

```diff
@@ -277,10 +54,34 @@
   }
 
 //panel elements
+  .move-properties {
+    table {
+      margin: 1rem auto;
+    }
+  }
+
+  .column-show {
+    margin-left: 40px;
+  }
+
   .field-log-in {
     color: #ead61c;
   }
 
+  .text-center {
+    text-align: center;
+  }
+
+  .panel-show {
+    background:rgba(255,255,255,0.15);
```

Comment:
Commas in function arguments should be followed by a single space<br>Colon after property should be followed by one space